test/dbus.c

```diff
@@ -654,18 +654,30 @@ TEST test_timeout_overflow(void)
         n_dbus = dbus_notification_new();
         n_dbus->app_name = "dunstteststack";
         n_dbus->app_icon = "NONE";
-        n_dbus->summary = "test_hint_urgency";
+        n_dbus->summary = "test_timeout_overflow";
         n_dbus->body = "Summary of it";
         n_dbus->expire_timeout = 2147484;
         gint64 expected_timeout = G_GINT64_CONSTANT(2147484000);
 
+        // Disable urgency_normal rule in order to avoid timeout override
```

Comment:
This shouldn't be needed right? The dbus timeout should overwrite the rule.

Reply:
> This shouldn't be needed right?

I did not know that the dbus timeout should have priority over the ones defined in the rules.

The change I made sets the notification timeout before applying the rules.
This gives priority to the timeouts defined in rules and not the one in the dbus.
That is why I had to change the test.

> The dbus timeout should overwrite the rule.

This makes it impossible for users to change a dbus timeout right?